libenkf/include/ert/enkf/site_config.h

```diff
@@ -101,6 +103,11 @@ typedef struct site_config_struct site_config_type;
   mode_t                   site_config_get_umask( const site_config_type * site_config );
 
   site_config_type       * site_config_alloc_empty();
+  site_config_type       * site_config_alloc_default();
+  site_config_type       * site_config_alloc_model_config(const char *);
+
+  config_content_type    * site_config_alloc_content(const site_config_type*, config_parser_type*);
```

Comment:
Two questions about this function:

1. Is it necessary to pass a `config_parser` instance? Possibly we could just create, use and discard the parser inside the function.

2. Is the function really required - at least public? I would guess that this could be integrated in the `site_config_alloc( const char * site_config_file)` constructor?